src/Sulu/Component/Webspace/Analyzer/RequestAnalyzer.php

```diff
@@ -45,11 +51,25 @@ public function analyze(Request $request)
     }
 
     /**
+     * Initilizes the request attributes lazy.
+     *
+     * @return RequestAttributes
+     */
+    protected function getAttributes()
+    {
+        if (!$this->attributes) {
```

Comment:
Which method? I am sure these methods can still be simplified into one, and still be used in combination with the preview.